tests/rptest/tests/prealloc_nodes.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2022 Vectorized, Inc.
+#
+# Use of this software is governed by the Business Source License
+# included in the file licenses/BSL.md
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0
+
+from rptest.tests.redpanda_test import RedpandaTest
+from ducktape.cluster.cluster_spec import ClusterSpec
+from ducktape.utils.util import wait_until
+from ducktape.tests.test import TestContext
+
+
+class PreallocNodesTest(RedpandaTest):
```

Comment:
When adding a generalization of existing code (in this case the node allocation done in the franzgoverifiabletest), it's a good practice to convert the existing code to use the generalized version, so that we don't have both floating around.  Can you do that in a followup PR please?

Reply:
Will do. I left that out here so I can keep this PR small (my PRs tend to get big).